front-end/css/stylesheet.css

```diff
@@ -0,0 +1,29 @@
+#map {
+    height: 100%;
+}
+
+/* Optional: Makes the sample page fill the window. */
+html, body {
+    height: 100%;
+    margin: 0;
+    padding: 0;
+}
+
+.carousel {
+    position: absolute;
+    display: block;
```

Comment:
Usually it's easier to read if you order css properties in lexicographical order :) (Or alphabetical, in this case)